### PR TITLE
Update the docs for publishing a pre-release

### PR DESCRIPTION
**Pull request by hannalaakso**
Ask users to run `npm ci` instead of `npm install`. Whilst `npm ci` will take a bit longer to run, it will not force any dependencies to update - at this point we just want to ensure that the users of this doc are using the dependencies specified in `package-lock.json`.

Also make the wording of checking out the pre-release branch more ambigious as the current command will create a new branch and this might not be appropriate.

Finally, add some instructions on how to update a previously published pre-release with new changes (@timpaul recently asked about this).

### CODE REVIEW

File: docs/releasing/publishing-a-pre-release.md [hunk 1]
```diff
@@ -18,11 +18,11 @@ Projects can point to this branch in their package.json, instead of to the publi
 
 ## Publish a pre-release
 
-1. Run `git checkout -b BRANCH-NAME` to switch to the branch you want to pre-release.
+1. Check out the Git branch you want to pre-release.
```
Comment (vanitabarrett) [inline docs/releasing/publishing-a-pre-release.md:21]: Would it be worth giving examples? For example:

`git checkout -b BRANCH-NAME` to checkout a new branch, or `git checkout BRANCH-NAME` to checkout an existing branch?

Comment (hannalaakso) [reply docs/releasing/publishing-a-pre-release.md:21]: Hmmm, we could but we need an optional extra step to make the code changes. Something like:

> 1. Run `git checkout -b BRANCH-NAME` to check out a new branch, or `git checkout BRANCH-NAME` to switch to the branch you want to pre-release.
> 2. Make the required changes and commit them.

Does this work @EoinShaughnessy?

Comment (EoinShaughnessy) [reply docs/releasing/publishing-a-pre-release.md:21]: @hannalaakso Yeah, looks good 👍🏻

Comment (hannalaakso) [reply docs/releasing/publishing-a-pre-release.md]: @vanitabarrett I've made the required changes if you have time to take another look.

**Review by vanitabarrett — APPROVED** [review]

**Pull request merged**